test/javascripts/acceptance/chat-live-pane-collapse-test.js

```diff
@@ -52,29 +80,72 @@ acceptance("Discourse Chat - Chat live pane", function (needs) {
     server.get("/chat/chat_channels/:chatChannelId", () =>
       helper.response({ chat_channel: { id: 1 } })
     );
+
+    server.post("/uploads/lookup-urls", () =>
+      helper.response([
+        200,
+        { "Content-Type": "application/json" },
+        [
+          {
+            url: "/images/avatar.png",
+          },
+        ],
+      ])
+    );
   });
 
   test("can collapse and expand youtube chat", async function (assert) {
-    const youtubeContainerSelector = ".lazyYT";
-    const close = ".chat-message-collapser-closed";
-    const open = ".chat-message-collapser-opened";
+    const youtubeContainerSelector = ".chat-message-container-1 .lazyYT";
+    const expandImage =
```

Comment:
```suggestion
    const expandImageSelector =
```

I'm wondering if this is clearer and will be more consistent with the variable above.

Reply:
Removed the `selector` here so I don't have to change the rest